README.md

```diff
@@ -107,4 +129,4 @@ $ nodemon bin/www
 
 ## Contributing
 
-See the [Contributing](contributing.md) guide to contribute to the project!
\ No newline at end of file
+See the [Contributing](contributing.md) guide to contribute to the project!
```

Comment:
Doesn't look like you meant to change this. Remove it from your PR.

Reply:
Hi @Jooms can we skip this change?

Reply:
Yea.